### PR TITLE
Feature/99 install presto theme

### PR DESCRIPTION
**Pull request by deneus**
Issue #99 
When you install the profile, you will arrive into presto_theme with all block positioned properly.

### CODE REVIEW

**Comment by deneus** [comment]
Merge with 8.x-1.x is done.

File: themes/presto_theme/config/optional/block.block.presto_theme_views_block__presto_product_listing_listing_block.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+langcode: en
```
Comment (chinthakagodawita) [inline themes/presto_theme/config/optional/block.block.presto_theme_views_block__presto_product_listing_listing_block.yml]: Remove this file, it's part of `presto_commerce`

File: themes/presto_theme/config/optional/block.block.presto_theme_contactblock.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+langcode: en
```
Comment (chinthakagodawita) [inline themes/presto_theme/config/optional/block.block.presto_theme_contactblock.yml]: Isn't this part of `presto_contact`?

File: themes/presto_theme/presto_theme.info.yml [hunk 3]
```diff
@@ -22,3 +22,7 @@ regions:
   footer: 'Footer'
   page_top: 'Page top'
   page_bottom: 'Page bottom'
+
+#libraries-override:
+#  bootstrap/tooltip: false
+#  bootstrap/popover: false
```
Comment (chinthakagodawita) [inline themes/presto_theme/presto_theme.info.yml]: Remove these lines

**Comment by deneus** [comment]
There is no reasons. 
The 3 requests are fixed.

**Comment by chinthakagodawita** [comment]
On hold for now, as discussed with @jrhodes-sitback, we should look into making this configurable instead:
* Checkbox on the 'configure presto' setp
* Checked by default
* Enable via batch tasks, similar to the eCommerce deps

**Comment by chinthakagodawita** [comment]
This has been obsoleted by #118.

This will be closed (and not merged) once there is a PR for #118 that includes the `*.yml` changes that are in this PR.

**Comment by chinthakagodawita** [comment]
Closing, see #131

**Pull request closed without merging**